mkdocs.yml

```
@@ -226,9 +226,11 @@ nav:
       - Overview: "apps/index.md"
       - Network to Code App Docs:
           - Nautobot ChatOps: "https://docs.nautobot.com/projects/chatops/en/latest/"
+          - Nautobot Device LifeCycle Management: "https://docs.nautobot.com/projects/device-lifecycle/en/latest/"
+          - Nautobot Device Onboarding: "https://docs.nautobot.com/projects/device-onboarding/en/latest/"
           - Nautobot Firewall Models: "https://nautobot-plugin-firewall-models.readthedocs.io/en/latest/"
-          - Nautobot Plugin Nornir: "https://docs.nautobot.com/projects/plugin-nornir/en/latest/"
           - Nautobot Golden Configuration: "https://docs.nautobot.com/projects/golden-config/"
+          - Nautobot Plugin Nornir: "https://docs.nautobot.com/projects/plugin-nornir/en/latest/"
           - Nautobot Single Source of Truth: "https://nautobot-plugin-ssot.readthedocs.io/en/latest/"
```

Should be working now:

```suggestion
          - Nautobot Single Source of Truth: "https://docs.nautobot.com/projects/ssot/en/latest/"
```